packages/2d/src/lib/components/TxtLeaf.ts

```diff
@@ -139,7 +139,7 @@ export class TxtLeaf extends Shape {
       -size.height / 2,
       bbox.width,
       bbox.height,
-    ).expand(lineWidth * miterLimitCoefficient);
+    ).expand([0, this.fontSize() * 0.5 * this.absoluteScale().x]).expand(lineWidth * miterLimitCoefficient);
```

Comment:
Looks like something broke in our prettier configuration. This line is longer than 80 characters and should be choped down:
```ts
    return new BBox(-size.width / 2, -size.height / 2, bbox.width, bbox.height)
      .expand([0, this.fontSize() * 0.5 * this.absoluteScale().x])
      .expand(lineWidth * miterLimitCoefficient);
```
Normally prettier would be run on commit, I'll see what's up with that.

Reply:
> Looks like something broke in our prettier configuration. This line is longer than 80 characters and should be choped down:
> 
> ```ts
>     return new BBox(-size.width / 2, -size.height / 2, bbox.width, bbox.height)
>       .expand([0, this.fontSize() * 0.5 * this.absoluteScale().x])
>       .expand(lineWidth * miterLimitCoefficient);
> ```
> 
> Normally prettier would be run on commit, I'll see what's up with that.

As it turns out, since TypeScript files are linted by ESLint and not Prettier. There's nothing wrong with the prettier configuration, it's just that the ESLint configuration doesn't have a rule restricting the maximum line length for TypeScript files.

I tried adding the following line to the `.eslintrc.json`, and upon rerunning `npx lint-staged` with a long line staged, an error was correctly thrown.

```diff
   "rules": {
     "require-yield": "off",
     "grouped-accessor-pairs": ["error", "getBeforeSet"],
     "eqeqeq": ["error", "always", {"null": "ignore"}],
     "curly": ["error", "multi-line"],
+    "max-len": ["error", {"code": 80}],
     "@typescript-eslint/explicit-member-accessibility": "error",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-non-null-assertion": "off",
```

![image](https://github.com/motion-canvas/motion-canvas/assets/10406383/0841df26-37c2-496c-8303-78704ed8ae2d)